### DISCUSSION TITLE
Angular SDK

### DISCUSSION THREAD

**Discussion by koddr** [Telegram-Mini-Apps/tma.js#441] [Ideas]
First, thanks for this library! It's really useful.

Do you have any plans to create an SDK for Angular? 
It would be great to have it, as we already have for React/Solid.

**Comment by heyqbnk** [comment]
Hey! Thanks. Surely, no. Maybe somebody will take care of it. I am currently working on the SDK version which will be easier to integrate into other similar libraries to Angular

**Reply by koddr** [reply]
Oh, okay. Thanks for reply 😉
